project-templates/import-pregnancies/import-pregnancies

```diff
@@ -0,0 +1,339 @@
+#!/usr/bin/env node
+
+var http = require('http'),
+    path = require('path'),
+    url = require('url'),
+    fs = require('fs');
+
+var csv = require('fast-csv'),
+    SELF = process.argv[1].split(path.sep).pop(),
+    flags = {};
+
+var usageInfo = `
+Usage:
+
+  COUCH_URL=http://admin:secret@localhost:5988 \\
+    ./${SELF} users.json contacts.json records.csv
+
+Options:
+
+  -d  Dry run, only prints request data.
+
+`;
+
+var usage = function() {
+  console.log(usageInfo);
+  process.exit(1);
+};
+
+if (!process.env.COUCH_URL || process.argv.length < 4) {
+  console.error("Define COUCH_URL");
+  usage();
+}
+
+if (process.argv.indexOf("-d") != -1) {
+  flags.dryrun = true;
+  process.argv.splice(process.argv.indexOf("-d"), 1);
+}
+
+var options = url.parse(process.env.COUCH_URL);
+
+if (options.protocol === 'https:') {
+  console.error('using https module');
```

Comment:
Because I wanted to avoid stdout because I was saving stdout for some reason that I can't remember exactly.